tests/integration_tests/functional/test_cpu_features_aarch64.py

```diff
@@ -82,11 +83,16 @@ def test_host_vs_guest_cpu_features_aarch64(uvm_nano):
     match cpu_model:
         case CpuModel.ARM_NEOVERSE_N1:
             assert host_feats - guest_feats == set()
-            # Kernel should hide this feature, but our guest kernel
-            # currently lacks the commit with this change.
-            # The commit that introduces the change:
-            # https://github.com/torvalds/linux/commit/7187bb7d0b5c7dfa18ca82e9e5c75e13861b1d88
-            assert guest_feats - host_feats == {"ssbs"}
+            # TODO: Remove "ssbs" once our guest kernel picks the following commit.
+            # https://github.com/torvalds/linux/commit/adeec61a4723fd3e39da68db4cc4d924e6d7f641
+            # See also https://github.com/firecracker-microvm/firecracker/issues/4898
```

Comment:
So for the feature to disappear, both host and guest kernel need the commit?

Reply:
yes. Currently only the host has the change that removes ssbs from the feature flags.